unittest/joint-configurations.cpp

```diff
@@ -46,6 +46,34 @@
 #include <boost/test/unit_test.hpp>
 
 
+Eigen::Quaterniond slerp(const Eigen::Quaterniond & q1, const Eigen::Quaterniond & q2,double t)
```

Comment:
I realized Eigen has an implementation of SLERP. We should maybe use `return q1.slerp (t, q2)`, no ?